core/src/main/java/com/adobe/granite/translation/connector/bootstrap/core/impl/BootstrapTranslationServiceImpl.java

```diff
@@ -231,7 +237,17 @@ public String createTranslationJob(String name, String description, String strSo
         String strTargetLanguage, Date dueDate, TranslationState state, TranslationMetadata jobMetadata)
         throws TranslationException {
         log.trace("BootstrapTranslationServiceImpl.createTranslationJob");
-        return bootstrapTmsService.createBootstrapTmsJob(name, strSourceLanguage, strTargetLanguage, dueDate);
+        String jobInfo = String.format("connector=aem,jobName=%s,srcLang=%s,trgLang=%s", name, strSourceLanguage, strTargetLanguage);
+        if (dueDate != null) {
+          String dueDateISO8601 = dueDate.toInstant().toString();
+          jobInfo = String.format("%s,dueDate=%s", jobInfo, dueDateISO8601);
+        }
+        try {
+          SourceFile[] sourceFiles = liltApiClient.getFiles(null);
+        } catch (Exception e) {
+          log.warn("error during createTranslationJob {}", e);
```

Comment:
Similarly, this doesn't seem like the right error handling.  I'm not sure how the bootstrap framework is set up to handle failure in general.  In the core we would often have a function like this return `Optional<String>` instead.  Re-throwing the exception after logging it might also work, depending on where that exception is caught.